source/deployment/sso-ldap.md

```diff
@@ -63,45 +63,52 @@ Yes, using the [bulk import tool](https://docs.mattermost.com/deployment/bulk-lo
 
 ##### How does deactivating users work? 
 
-Users can be deactivated as follows:
+When LDAP authentication is used in Mattermost, user deactivation must be done via the LDAP server. 
 
-- LDAP Server
-  - Delete the user (they will be deactivated in Mattermost on the next sync)
-  - Deselect the user in the user filter
-- Active Directory
-  - Run the following query in the user filter: `(&(objectCategory=Person)(!(UserAccountControl:1.2.840.113556.1.4.803:=2)))`
+There are two main ways to do this: 
+
+1. User Deletion: If the user is completely removed from the LDAP server, they will be deactivated in Mattermost on the next synchronization. 
+2. User Filter: Set the [user filter](https://docs.mattermost.com/administration/config-settings.html#user-filter) to only select the subset of LDAP users you want to have access to Mattermost. When someone is removed from the selected group, they will be deactivated in Mattermost on the next synchronization. 
+
+For Active Directory, to filter out deactivated users you must set the user filter to `(&(objectCategory=Person)(!(UserAccountControl:1.2.840.113556.1.4.803:=2)))`.  
 
 ##### Can I connect to multiple Active Directory servers? 
 
-No, this is not possible at the moment.  You will need to connect the instances in a forest.
+There is currently no built-in way to connect to multiple AD servers. You will need to connect the instances in a forest before connecting to Mattermost.
 
 Consider upvoting the [feature request](https://mattermost.uservoice.com/forums/306457-general/suggestions/13589904-add-the-abilitiry) on our forum.
 
-##### AD/LDAP not available on this server
+##### When trying to sign in, I see the error `AD/LDAP not available on this server`
+
+This indicates that there is a problem somewhere with your configuration. We recommend that you: 
+1. Confirm Enterprise Edition is installed
+2. Check your license to confirm it's uploaded and not expired 
+3. Check your Mattermost configuration settings to ensure that LDAP is enabled, and the settings are correct
```

Comment:
One more thing to check is to click the "Run AD/LDAP Test" button to test whether Mattermost can connect to the LDAP server successfully.